sdk/cosmos/azure-cosmos-spark_3-1_2-12/Samples/Python/NYC-Taxi-Data/01_Batch.ipynb

```diff
@@ -323,7 +323,7 @@
     "  \"spark.cosmos.accountKey\": cosmosMasterKey,\n",
     "  \"spark.cosmos.database\": \"SampleDatabase\",\n",
     "  \"spark.cosmos.container\": \"GreenTaxiRecords\",\n",
-    "  \"spark.cosmos.read.partitioning.strategy\": \"Default\",\n",
+    "  \"spark.cosmos.read.partitioning.strategy\": \"Restrictive\",#IMPORTANT - any other partitioning strategy will result in indexing not being use to count - so latency and RU would spike up\n",
```

Comment:
will the situation change in future? 
should we always recommend customer to use "restrictive" now?

Reply:
Just when using custom queries with aggregates like COUNT(0). Query team won't be able to use index - instead this becomes a full table scan if the EPK range doesn't align with a physical partition.
Need to get details from Query team - but our sample should for the "efficient count" sample definitely avoid the table scan